src/hevm/src/EVM/Traversals.hs

```diff
@@ -323,6 +323,15 @@ mapExprM f expr = case expr of
   IllegalOverflow a -> do
     a' <- mapM (mapPropM f) a
     f (IllegalOverflow a')
+  InvalidMemoryAccess a -> do
```

Comment:
Shall we add these cases to `foldExpr` in the same file?

Reply:
Ah, good catch, I didn't see it! Fixed in new push :)